File: app/src/main/java/ir/ahfz/rentcar/io/network/model/FuelResponse.java
package ir.ahfz.rentcar.io.network.model;

import java.util.List;

import ir.ahfz.rentcar.io.network.BaseResponse;

public class FuelResponse extends BaseResponse {

    private List<Fuel> fuels = null;

    public List<Fuel> getFuels() {
        return fuels;
    }

    public void setFuels(List<Fuel> fuels) {
        this.fuels = fuels;
    }

    public class Fuel {

        private int id;
        private String fuel;
        private String createdAt;
        private String updatedAt;

        public int getId() {
            return id;
        }

        public void setId(int id) {
            this.id = id;
        }

        public String getFuel() {
            return fuel;
        }

        public void setFuel(String fuel) {
            this.fuel = fuel;
        }

        public String getCreatedAt() {
            return createdAt;
        }

        public void setCreatedAt(String createdAt) {
            this.createdAt = createdAt;
        }

        public String getUpdatedAt() {
            return updatedAt;
        }

        public void setUpdatedAt(String updatedAt) {
            this.updatedAt = updatedAt;
        }

    }
}

File: app/src/main/java/ir/ahfz/rentcar/io/network/model/FuelResponse.kt
package ir.ahfz.rentcar.io.network.model

import ir.ahfz.rentcar.io.network.BaseResponse

/**
 * Data model for Fuel response
 */
data class FuelResponse(
    var fuels: List<Fuel>? = null
) : BaseResponse() {

    /**
     * Data model for Fuel
     */
    data class Fuel(
        var id: Int = 0,
        var fuel: String? = null,
        var createdAt: String? = null,
        var updatedAt: String? = null
    )
}
